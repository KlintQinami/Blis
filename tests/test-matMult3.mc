int main() {
  float scalar = 1337.;
  mat1x4 colvec = mat1x4(1., 2., 3., 4.);
  mat1x4 prod2 = colvec * scalar;
  mat4x1 rowvec = mat4x1(1., 2., 3., 4.);
  mat4x1 prod = scalar * rowvec;
  printf(prod.x);
  printf(prod.y);
  printf(prod.z);
  printf(prod.w);
  printf(prod2.x);
  printf(prod2.y);
  printf(prod2.z);
  printf(prod2.w);
  return 0;
}
