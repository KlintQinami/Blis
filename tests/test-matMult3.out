1337.000000
2674.000000
4011.000000
5348.000000
1337.000000
2674.000000
4011.000000
5348.000000
